gdal/ogr/ogrsf_frmts/mssqlspatial/drv_mssqlspatial.html

```diff
@@ -139,6 +139,7 @@ <h2>Configuration options</h2>
     <li><b>MSSQLSPATIAL_SHOW_FID_COLUMN</b>: Force to display the FID colums as a feature attribute. Default = "NO".</li>
     <li><b>MSSQLSPATIAL_USE_GEOMETRY_COLUMNS</b>: Use/create geometry_columns metadata table in the database. Default = "YES".</li>
     <li><b>MSSQLSPATIAL_LIST_ALL_TABLES</b>: Use mssql catalog to list available layers. Default = "NO".</li>
+    <li><b>MSSQLSPATIAL_USE_GEOMETRY_VALIDATION</b>: (From GDAL 2.5.0) Let the driver to detect the geometries which would trigger run time errors at MSSQL server. The driver tries to correct these geometries before submitting that to the server. Default = "NO".</li>

```

Comment:
the 'to' after 'Let the driver' should be remove, no ?

Reply:
probably yes, will fix that